www/modules/contrib/devel/log.json

```diff
@@ -0,0 +1,51 @@
+{"level":"debug","msg":"main.go started","time":"2017-03-24T19:33:53+01:00"}
```

Comment:
probably shouldn't commit this file? (apologies if you already got rid of it, just going through commits)